code/game/objects/items/ego_weapons/special.dm

```diff
@@ -144,6 +144,8 @@
 	AddElement(/datum/element/update_icon_updates_onmob)
 
 /obj/item/ego_weapon/iron_maiden/proc/Multihit(mob/living/target, mob/living/user, attack_amount)
+	var/userjust = (get_modified_attribute_level(user, JUSTICE_ATTRIBUTE))
```

Comment:
This is a shit way of doing this.
Stop doing this.
Just call Attack()

Reply:
I tested it and doing that just causes it to infinitely loop